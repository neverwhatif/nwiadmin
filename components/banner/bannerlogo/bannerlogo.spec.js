import React from 'react';
import { shallow } from 'enzyme';

import BannerLogo from './index';

jest.mock('nwiadmin/components/link', () => 'Link');

describe('BannerLogo', () => {
    it('should render', () => {
        expect(shallow(<BannerLogo />)).toMatchSnapshot();
    });
});
