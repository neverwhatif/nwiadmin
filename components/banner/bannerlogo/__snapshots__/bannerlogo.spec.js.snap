// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerLogo should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BannerLogo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Link
        className="link"
        to="/"
      >
        <Logo
          className="img"
          isReverse={true}
        />
      </Link>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Logo
          className="img"
          isReverse={true}
        />,
        "className": "link",
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "className": "img",
          "isReverse": true,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "Link",
    },
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Link
          className="link"
          to="/"
        >
          <Logo
            className="img"
            isReverse={true}
          />
        </Link>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Logo
            className="img"
            isReverse={true}
          />,
          "className": "link",
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "className": "img",
            "isReverse": true,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "Link",
      },
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
